docs/notes.md

```diff
@@ -1,3 +1,19 @@
+## Costing
```

Comment:
Better move it into Cost.md (already exists) or new-costing.md

Reply:
Done